modules/newScores.js

```diff
@@ -1,24 +1,24 @@
 import Score from './constructor.js';
-import { inputName, inputScore, form } from './elements.js';
-import { createScores, storage } from './storage.js';
+import { form, inputName, inputScore } from './elements.js';
+import { storage } from './storage.js';
 import addScore from './addScore.js';
 
-export default function newScore(event) {
+export default async function newScore(event) {
```

Comment:
- [ ] Kindly change this to an arrow function as the previous reviewer suggested.